adapters/admixer/admixertest/exemplary/optional-params.json

```diff
@@ -44,6 +44,29 @@
             }
           }
         }
```

Comment:
Can you please also add a test where both the imp `BidFloor` and `CustomBidFloor` are set to non-zero values and the request to your service has it set to the imp's `BidFloor` value? You can just do this by adding `imp.bidfloor` in one of the imps above and modifying the expected HTTP request accordingly as well